pkgs/by-name/ge/gema/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  installShellFiles,
+}:
+
+stdenv.mkDerivation rec {
```

Nixpkgs is moving away from use of `rec` towards `finalAttrs` for `mkDerivation`:

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

---

Thank you! Change made.